docker/localhost.docker-compose.yaml

```diff
@@ -72,8 +72,8 @@ services:
           - cvm
           - nginx_cvm
     restart: unless-stopped
-    profiles: [ "all", "nginx", "cvm", "chem", "jupyter_notebook", "notebook", "scripting",
```

Comment:
"jupyter_notebook", "notebook", "jupyter_kernel_gateway", "jkg" are still very useful profiles, lets use them in addition to jupyter